test/test_bundle_importer.js

```diff
@@ -177,13 +197,15 @@ describe('Bundle Importer', function() {
       bundleImporter._execute_deploy = executor;
       bundleImporter._execute_addUnit = executor;
       bundleImporter._execute_addRelation = executor;
+      bundleImporter._execute_setAnnotations = executor;
 
       var sortedRecords = [
         { method: 'addCharm' },
         { method: 'addMachines' },
         { method: 'deploy' },
         { method: 'addUnit' },
-        { method: 'addRelation' }
+        { method: 'addRelation' },
+        { method: 'setAnnotations' }
```

Comment:
just addCharm add service is just called 'deploy'. In the future we will also want to add addCharm functionality to the environment instead of relying on 'deploy'. Right now we call the fakebackend directly to support it in the sandbox mode.